File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/cdp/di/ecs/model/retailers/StoreEntity.java
/**
 * (C) Koninklijke Philips N.V., 2015.
 * All rights reserved.
 */
package com.philips.cdp.di.ecs.model.retailers;

import java.io.Serializable;

public class StoreEntity implements Serializable{

    private static final long serialVersionUID = -4549397314749988036L;
    private String name;
    private String availability;
    private String isPhilipsStore;
    private String philipsOnlinePrice;
    private int logoHeight;
    private int logoWidth;
    private String buyURL;
    private String logoURL;
    private String xactparam;

    public String getName() {
        return name;
    }

    public String getAvailability() {
        return availability;
    }

    public String getIsPhilipsStore() {
        return isPhilipsStore;
    }

    public String getPhilipsOnlinePrice() {
        return philipsOnlinePrice;
    }

    public int getLogoHeight() {
        return logoHeight;
    }

    public int getLogoWidth() {
        return logoWidth;
    }

    public String getBuyURL() {
        return buyURL;
    }

    public String getLogoURL() {
        return logoURL;
    }

    public String getXactparam() {
        return xactparam;
    }
}

File: Source/ecs/Source/Library/philipsecommercesdk/src/main/java/com/philips/cdp/di/ecs/model/retailers/StoreEntity.kt
/**
 * (C) Koninklijke Philips N.V., 2015.
 * All rights reserved.
 */
package com.philips.cdp.di.ecs.model.retailers

import java.io.Serializable

class StoreEntity : Serializable {
    val name: String? = null
    val availability: String? = null
    val isPhilipsStore: String? = null
    val philipsOnlinePrice: String? = null
    val logoHeight: Int = 0
    val logoWidth: Int = 0
    val buyURL: String? = null
    val logoURL: String? = null

    companion object {

        private const val serialVersionUID = -4549397314749988036L
    }
}
